packages/veritone-react-common/src/components/TranscriptSearchModal/index.js

```diff
@@ -67,7 +67,7 @@ export default class TranscriptSearchModal extends React.Component {
 export const TranscriptSearchForm = ( { defaultValue, cancel, onSubmit, onChange, onKeyPress, inputValue } ) => {
   return (
   <div>
-    <DialogTitle>Search Transcript</DialogTitle>
+    <DialogTitle>Search by Keyword</DialogTitle>
```

Comment:
I'm not convinced we shouldn't take this opportunity to "generalize" this stuff.

Let's leave this for now.  But at the end of this I'll probably take a pass through here and tweak the copy to be more in line with our generalized, non-media-specific data model.

Reply:
FYI, just recapping our last conversation but most of that information is defined here and will be passed into the SearchBar to configure it.

See the endpoint: https://enterprise.stage.veritone.com/api/engine/category?time=1517268957867